src/components/Chat/InputMessageV2.tsx

```diff
@@ -144,6 +147,15 @@ const InputMessageV2 = ({
   }, [isDisableButton()]);
 
   const toggleChnage = () => {
+    if (messageDisable) {
+      ToastMessage.show({
+        type: 'asNative',
+        text1: messageDisable,
+        position: 'bottom'
+      });
+      return;
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This block of code checks if `messageDisable` is provided and, if so, displays a toast message and returns early. This is a good way to provide immediate feedback to the user when a functionality is disabled.
```diff
+     if (messageDisable) {
+       ToastMessage.show({
+         type: 'asNative',
+         text1: messageDisable,
+         position: 'bottom'
+       });
+       return;
+     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->